cmd/mattermost/commands/message_export.go

```diff
@@ -15,21 +15,48 @@ import (
 )
 
 var MessageExportCmd = &cobra.Command{
-	Use:     "export",
-	Short:   "Export data from Mattermost",
-	Long:    "Export data from Mattermost in a format suitable for import into a third-party application",
-	Example: "export --format=actiance --exportFrom=12345",
-	RunE:    messageExportCmdF,
+	Use:   "export",
+	Short: "Export data from Mattermost",
+	Long:  "Export data from Mattermost in a format suitable for import into a third-party application",
+}
+
+var ScheduleExportCmd = &cobra.Command{
+	Use:     "schedule",
+	Short:   "Schedule a Export data job in Mattermost",
+	Long:    "Schedule a Export data job in Mattermost (this will use the workers to run the export)",
+	Example: "export schedule --format=actiance --exportFrom=12345 --timeoutSeconds=12345",
+	RunE:    scheduleExportCmdF,
+}
+
+var CsvExportCmd = &cobra.Command{
+	Use:     "csv",
+	Short:   "Export data from Mattermost in CSV format",
+	Long:    "Export data from Mattermost in CSV format (this will export the data directly)",
```

Comment:
Not sure that we need to specify that this "will export the data directly". Thoughts?